src/main/java/dekk/pw/pokemate/Config.java

```diff
@@ -37,13 +38,11 @@
     private static boolean eggsHatching;
     private static boolean transferPrefersIV;
 	private static int cpMinimumForMessage;
-    public final static String POKE = "Pok\u00E9";
-
     private static Properties properties = new Properties();
 
-    static {
+    public static void load(String configPath) {
         try {
-            properties.load(new FileInputStream("config.properties"));
+            properties.load(new FileInputStream(configPath));
```

Comment:
putting the config path into the config is rather counter intuitive....

If you are loading a config to find the path to the config you are accomplishing nothing...